include/ignition/common/detail/SpecializedPlugin.hh

```diff
@@ -19,7 +19,6 @@
 #define IGNITION_COMMON_DETAIL_SPECIALIZEDPLUGIN_HH_
 
 #include <ignition/common/config.hh>
-#include "ignition/common/SpecializedPlugin.hh"
```

Comment:
I think this can be retained if its header guards are restored

Reply:
54cbbac4